src/main/java/ui/UI.java

```diff
@@ -150,6 +155,7 @@ private void disableUI(boolean disable) {
     }
 
     private void showMainWindow(String repoId) {
+        boolean isAFirstTimeUser = logic.getStoredRepos().isEmpty();
```

Comment:
I thought I asked this line to be moved down? There is no point setting this variable until we need it, which happens at line 183.